sections/semantics-sections.include

```diff
@@ -1049,15 +1049,15 @@
     </dd>
   </dl>
 
-  The <{header}> element <a>represents</a> introductory content for its nearest ancestor <{main}> 
+  The <{header}> element <a>represents</a> introductory content for its nearest ancestor <{main}>
   element or <a>sectioning content</a> or <a>sectioning root</a> element. A <{header}> typically
   contains a group of introductory or navigational aids.
 
-  When a <{header}> element's nearest ancestor <a>sectioning root</a> element is the <{body}> element, 
-  and it is not a descendant of the <{main}> element or a <a>sectioning content</a> element, 
-  then that <{header}> is scoped to the <{body}> element and represents introductory content for the page as a whole.
+  When a <{header}> element's nearest ancestor <a>sectioning root</a> element is the <{body}> element,
+  and it is not a descendant of the <{main}> element or a <a>sectioning content</a> element,
+  then that <{header}> is scoped to the <{body}> element and represents mostly site-oriented content, or introductory content for the page as a whole.
```

Comment:
mostly? It isn't clear if you mean "should represent X, may represent Y", or "should be predominantly about X and a bit about Y", or if the word needs to be there at all.

Not a show-stopper as far as I can tell, just editorial preference.